ui/components/app/assets/asset-list/network-filter/network-filter.tsx

```diff
@@ -85,19 +89,20 @@ const NetworkFilter = ({ handleClose }: SortControlProps) => {
     handleClose();
   };
 
-  useEffect(() => {
-    const testnetChains: string[] = TEST_CHAINS;
-    const mainnetChainIds = Object.keys(allNetworks || {}).filter(
-      (chain) => !testnetChains.includes(chain),
-    );
-    setChainsToShow(mainnetChainIds);
-  }, []);
-
   const allOpts: Record<string, boolean> = {};
   Object.keys(allNetworks || {}).forEach((chain) => {
     allOpts[chain] = true;
   });
 
+  const allAddedPopularNetworks = FEATURED_NETWORK_CHAIN_IDS.filter(
```

Comment:
We could choose to pull names from `networkConfigurationsByChainId`, in case the user has chosen to give their network some non canonical name, but I think our constants are okay.

Reply:
`NETWORK_TO_NAME_MAP` would be another place to pull them via chain id instead of the .find

Reply:
You're right, I'll make a change to use networkConfigurationsByChainId